README.md

```diff
@@ -5,7 +5,19 @@ Talkable Ruby Gem to make your own referral program in Sinatra or Rails applicat
 
 ## Demo
 
-Take a spree demo app and intall Talkable
+Example of usage at http://github.com/talkable/talkable-spree-example
+
+Live demo available at http://spree-example.talkable.com
+
+## Requirements
+
+Gem requires:
+ - `Ruby` version since `1.9.3`
+ - `Rack` version since `1.5.2`
+
+Gem supports:
+ - `Ruby on Rails` since `4.0.0`
+ - `Sinatra` since `1.4.0`
 
 ## Intallation
```

Comment:
Let's describe a kind of plan here:

1. Generate the configuration file (*)
2. Middleware installation (*)
3. Post-Checkout Integration 
4. Floating widget integration (*)
4. /invite integration (*)

* - Automated in Ruby On Rails using a generator